Fixed some typos in readme.md (#190)
aquasecurity/trivy: README.md

Then, those vulnerabilities will be saved on [vuln-list](https://github.com/aquasecurity/vuln-list/tree/master/alpine).

`alpine-secdb` has 6959 vulnerabilities (as of 2019/05/12).
`vuln-list` has 11101 vulnerabilities related to Alpine Linux (as of 2019/05/12).
There is a difference in detection accuracy because the number of vulnerabilities is nearly doubled.

> Besides, `Trivy` analyzes the middle layer as well and find out which version of the library was used for static linking.

Reviewer: I actually prefer "In addition" here.

> Besides, `Trivy` analyzes the middle layer as well and find out which version of the library was used for static linking.

Reviewer: ```suggestion
In addition, `Trivy` analyzes the middle layers as well to find out which version of the library was used for static linking.
```

## vs Quay, Docker Hub, GCR

As `Quay` seems to use `Clair` internally, it has the same accuracy as `Clair`. `Docker Hub` can scan only official images. `GCR` hardly detects vulnerabilities on Alpine Linux. Also, it is locked to a specific registry.

> `Trivy` can be used regardless of the registry. Besides, it is easy to be integrated with CI/CD services.

Reviewer: ```suggestion
`Trivy` can be used regardless of the registry, and it is easily integrated with CI/CD services.
```
